runtime/test/trace/src/TracerTests.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Lucas Satabin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package swam
+package runtime
+package trace
+
+import utest._
+
+object TracerTests extends TestSuite {
```

Comment:
Is that a leftover from another change? I thought we already had this kind of tests.

Reply:
It seems so ... to remove then :)